evalml/tests/component_tests/test_components.py

```diff
@@ -568,7 +569,7 @@ def test_default_parameters(cls):
     assert cls.default_parameters == cls().parameters, f"{cls.__name__}'s default parameters don't match __init__."
 
 
-@pytest.mark.parametrize("cls", [cls for cls in all_components() if cls not in [StackedEnsembleRegressor, StackedEnsembleClassifier]])
+@pytest.mark.parametrize("cls", [cls for cls in all_components() if cls not in [StackedEnsembleRegressor, StackedEnsembleClassifier, ARIMARegressor]])
```

Comment:
I don't think ARIMARegressor needs to be in the list of excluded component classes, i.e. the arima default parameters should not raise a warning.